forms-flow-service/src/resourceBundles/en/resourceBundles.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable max-len */
-export const RESOURCE_BUNDLES_EN = {
+export const RESOURCE_BUNDLES_EN ={
```

Comment:
Should this be changed to RESOURCE_BUNDLES_DE ?
Just to confirm

Reply:
no this resource bundle related to English right